newsletter-november-2022.rst

```diff
@@ -0,0 +1,97 @@
+
+- :doc:`sphinx-rtd-theme 1.1.0 </theme-release-110>` has been released with minor improvements and as a big step towards future releases with larger changes.
+- After rolling out auto-cancelling builds `last month <https://blog.readthedocs.com/cancel-old-builds/>`_, we have already recorded a whopping 10% decrease in builds.
+  We're really happy with how this turned out, not least that it effectively reduces our cloud footprint by 10% 🌱.
+- We started incrementally refactoring our own documentation to match the `Diátaxis framework <https://diataxis.fr/>`_. We will be writing more about this, as we are currently gaining practical experience.
+- Much thanks to `AA-Turner <https://github.com/AA-Turner>`_ and the Sphinx community for working together on a proposal and releasing the extension `sphinxcontrib-jquery <http://pypi.org/project/sphinxcontrib-jquery>`_. This extension is required for themes and extensions that need jQuery from Sphinx 6 and onewards. Sphinx 6.0 is scheduled for December 2022 and will no longer bundle jQuery.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+Upcoming features
+-----------------
+
+- A 1.2.0 release of :ref:`sphinx-rtd-theme <sphinx_rtd_theme110_upcoming_releases>` will support docutils 0.18 and Sphinx 6.
+- We're working on improving our integration with Material for MkDocs, which is a great theme for MkDocs documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
+- 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+
+Possible issues
+---------------
+
+If you find regressions in any new releases of the `sphinx-rtd-theme <https://sphinx-rtd-theme.readthedocs.io/>`_,
+please don't hesitate to `open an issue on GitHub <https://github.com/readthedocs/sphinx_rtd_theme/>`_.
+
+We continue planning to be more active in deprecating old and outdated approaches to using our platform in Q4.
+We don't have anything firm to announce here yet,
+but we do plan to be more active in removing these features in the coming months.
+
+
+Tip of the month
+----------------
+
+This Tip of the Month comes from our own experience. We have greatly benefitted from the following two Python/Django projects:
+
+- `blacken-docs <https://github.com/adamchainz/blacken-docs>`_ is a pre-commit linter that checks and formats your Python code. In other words, it helps you make sure that your documentation's code examples look great.
+- `django-upgrade <https://github.com/adamchainz/django-upgrade>`_ saves us a lot of time each time we upgrade to the latest Django version.
+
+As you might have noticed, both of these projects are maintained by `Adam Johnson <https://adamj.eu/>`_. Thanks, Adam 👋
```

Comment:
@ericholscher @humitos I was trying to think about Tips from a "what are we finding useful" approach.

Is that fine? I'm opening up a newsletter PR for December 2022 very soon so we can already start thinking about the next tips etc :)

Reply:
I like this section, yeah! I image it will be useful to promote Sphinx extensions as well 💯

Reply:
Yea, this section is great. I think we should also tweet this.

Reply:
Thanks! I've added a reminder for myself to draft a tweet once this one us published (since it could potentially link to the post)